systemtest/src/test/java/io/strimzi/systemtest/UserST.java

```diff
@@ -151,6 +153,18 @@ void createBigAmountOfUsers(String typeOfUser) {
         }
     }
 
+    private void deployTestSpecificResources() {
+        testClassResources().kafka(testClassResources().defaultKafka(CLUSTER_NAME, 1, 1)
```

Comment:
What change the size of the cluster from 3 to 1?

Reply:
This tests are mostly focused on User Operator so I wanted to save some resources during the test - 3 kafka pods instead of 1, test itself should not be affected by it.